exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -179,6 +179,26 @@ drill.exec: {
     // Use plain Java compilation where available
     prefer_plain_java: false
   },
+  spill: {
```

Comment:
Please modify distribution/resources/drill-override-example.conf to explain how to set up spilling. Add a comment (see below) about deprecating the sort-specific config, but that it will still work for a few more releases.

This file (drill-module.conf) is visible only to developers. drill-override.conf is the file users modify. drill-override-example.conf is the example "starter" file that new users start with.